### PR TITLE
Feature read

### PR DESCRIPTION
**Pull request by Ben-21**

### CODE REVIEW

File: frontend/src/components/ProjectCard.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    );
+}
+
+const StyledCard = styled(Card)`
+  && {
+    max-width: 345px;
+    background-color: grey;
+    margin-top: 16px;
+  }
+`;
```
Comment (Pia0912) [inline frontend/src/components/ProjectCard.tsx:48]: This mean red icon shows that you are missing an empty line at the end of this file 😃

File: frontend/src/pages/Gallery.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 1.1em;
+`;
+
+const ProjectCardWrapper = styled.div`
+  min-width: 380px; /* Set the desired fixed width here */
+`;
```
Comment (Pia0912) [inline frontend/src/pages/Gallery.tsx:34]: again a missing empty line 😃

**Review by Pia0912 — APPROVED** [review]
Really Great Work! 🚀 especially your tests look really awesome! 🔝

**Comment by Ben-21** [comment]
> Really Great Work! 🚀 especially your tests look really awesome! 🔝

Thank you, I appreciate 🙂
Also, always forgetting the empty line

**Pull request merged**